### PR TITLE
feat(core) Enable account level SCP mgmt

### PR DESCRIPTION
**Pull request by charliejllewellyn**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

This change adds an optional key on mandatory and workload accounts. This is so account level SCPs can be managed from the installer. If the key “scps”: [ “SCP policy 1”, “SCP policy 2” ] is set the installer will add the policies to the account. If the user removes policies from the config but leaves the key in place e.g. “scps”: [ “SCP policy 1” ] or “scps”: [] the policies on the accounts will be updated to exactly match policies defined in the config. If the key is completely removed from the config then the policies will remain in place as it assumed the user wants to manage the policies outside the installer.

Tested scenarios:
Adding:
  - single mandatory account single SCP - PASS
  - single mandatory account multiple SCPs - PASS
  - single workload account multiple SCPs - PASS
  - multiple mandatory account multiple SCPs - PASS
Removing:
  - single mandatory account single SCP - PASS
  - single workload account multiple SCPs - PASS
  - multiple mandatory account single SCP - PASS
  - multiple mandatory account multiple SCPs - PASS
  - completely removing key from file - PASS (leaves all SCPs on the account in place - assuming the user wishes to change to managing SCPs outside of the installer)
SCP restoration:
  - Attempted manual removal of policy from mandatory account via the UI - PASS (In this case the policy is reattached to the account)
  - Attempted manual removal of policy from workload account via the UI - PASS (In this case the policy is reattached to the account)
  -  Attempted manual update of policy set on single mandatory account via the UI - PASS (In this case the policy is restored to the original policy)

### CODE REVIEW

File: src/core/runtime/src/add-scp-step.ts [hunk 1]
```diff
@@ -103,6 +103,13 @@ export const handler = async (input: AddScpInput) => {
     acceleratorPrefix,
   });
 
+  await scps.attachOrDetachPoliciesToAccounts({
```
Comment (naveenkoppula) [inline src/core/runtime/src/add-scp-step.ts:106]: We are performing add-scps, when an scp is modified in backgroud using cloudwatch event. Need to add same funciton call in 
``` accelerator/src/deployments/runtime/src/ou-validation-events/policy-changes.ts ```

Comment (charliejllewellyn) [reply src/core/runtime/src/add-scp-step.ts:106]: I just tested this by editing the SCP attached to a single account (i.e. not attached to any OU's) with a non accelerator role. The change altered one of the statements from "deny" to "allow". In this scenario the SCP policy change was caught and reinstated as it should be. Not sure that the change you mention above is required unless I have miss understood?

Comment (naveenkoppula) [reply src/core/runtime/src/add-scp-step.ts:106]: yeah. You are correct we are already performing policy changes in generic manner.

File: src/deployments/runtime/src/ou-validation-events/policy-changes.ts [hunk 2]
```diff
@@ -155,6 +155,12 @@ export const handler = async (input: PolicyChangeEvent) => {
       acceleratorOus: config.getOrganizationalUnits(),
       acceleratorPrefix,
     });
+
+    await scps.attachOrDetachPoliciesToAccounts({
```
Comment (naveenkoppula) [inline src/deployments/runtime/src/ou-validation-events/policy-changes.ts:159]: Not required. 👎

Comment (charliejllewellyn) [reply src/deployments/runtime/src/ou-validation-events/policy-changes.ts]: Ace - just pushing an update with the changes removed. Once it's passed testing should be good. Thanks for the review :)

**Review by naveenkoppula — APPROVED** [review]

**Pull request merged**